src/main/java/bio/terra/service/dataset/flight/ingest/CombinedFileIngestOptionalStep.java

```diff
@@ -15,7 +16,16 @@ public boolean isEnabled(FlightContext context) {
   }
 
   @Override
-  public String getReason() {
+  public String getSkipReason() {
     return "there are no files to ingest";
   }
+
+  @Override
+  public String getRunReason(FlightContext context) {
+    long numFiles =
+        Objects.requireNonNullElse(
+            context.getWorkingMap().get(IngestMapKeys.NUM_BULK_LOAD_FILE_MODELS, Long.class),
+            0L);
+    return numFiles + "bulk load files found to ingest";
```

Comment:
you need an extra space here.  I'd recommend using string format. e.g.
```suggestion
    return String.format("%s bulk load file(s) found to ingest", numFiles);
```